### ISSUE TITLE
Compatibility with python3

### ISSUE THREAD

**Issue by kungfupanda792** [brennerm/aws-quota-checker#12]
Getting the following error while running it inside a kubernetes pod or even as a container

Traceback (most recent call last):
  File "/usr/local/bin/aws-quota-checker", line 7, in <module>
    from aws_quota.cli import cli
  File "/usr/local/lib/python3.7/site-packages/aws_quota/cli.py", line 2, in <module>
    from aws_quota.utils import get_account_id
  File "/usr/local/lib/python3.7/site-packages/aws_quota/utils.py", line 6, in <module>
    def get_account_id(session: boto3.Session) -> str:
  File "/usr/lib64/python3.7/functools.py", line 490, in lru_cache
    raise TypeError('Expected maxsize to be an integer or None')
TypeError: Expected maxsize to be an integer or None

**Comment by brennerm** [comment]
Hey @kungfupanda792, thanks for reporting. Seems like that's an Python 3.7 issue cause the functools interface changed a bit in Python 3.8.

Will fix it in the next days. Until then you can use Python 3.8 if that's an option for you.

**Comment by brennerm** [comment]
Issue should be fixed with version 1.4.1.

BTW here are the changes that were necessary. ;)
```diff
-@functools.lru_cache                                                                                                                                                                                                            
+@functools.lru_cache()
```
